src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -219,7 +221,29 @@ private static boolean isToolTip(Node currentNode) {
         
         return false;
     }
-
+    
+    private static boolean isMotdComponent(Node currentNode) {      
+        if (currentNode.getNodeName().equalsIgnoreCase("script")) {
+            return currentNode.getTextContent().contains("closeMotd");
+            
+        } else if (currentNode.getNodeName().equalsIgnoreCase("div")) {
+            NamedNodeMap attributes = currentNode.getAttributes();
+            
+            if (attributes == null) { 
+                return false;
+            }
+                
+            for (int i = 0; i < attributes.getLength(); i++) {
```

Comment:
It doesn't work. Can only iterate over an array or an instance of java.lang.Iterable